### PR TITLE
Added Reboot button at the end of the installation

### PR DESCRIPTION
**Pull request by teclator**
## Problem

When the installation finish there is an option for restarting the installation but there is no option for rebooting the system as there is no panel or topbar menu with the poweroff options.

- https://trello.com/c/fDPinGTk/2917-2-d-installer-add-a-reboot-button

## Solution

- Added a **reboot** system button.

## Screenshots

Reboot

### CODE REVIEW

File: web/src/Installer.jsx [hunk 1]
```diff
@@ -80,6 +80,8 @@ function Installer() {
       dispatch({ type: "SET_DBUS_ERROR", payload: { error: "Connection lost" } });
     });
   }, []);
+  
+  return <InstallationFinished />;
```
Comment (imobachgs) [inline web/src/Installer.jsx:84]: I guess is it just a leftover.

File: web/src/lib/client/manager.test.js [hunk 2]
```diff
@@ -60,3 +63,16 @@ describe("#startInstallation", () => {
...
+describe("#rebootSystem", () => {
+  beforeEach(() => {
+      cockpit.spawn = jest.fn().mockResolvedValue("");
+  });
+
+  it("returns whether the system reboot command was called or not", async () => {
+    const client = new ManagerClient(dbusClient);
+    const reboot = await client.rebootSystem();
+    expect(cockpit.spawn).toHaveBeenCalledWith(["/usr/sbin/shutdown", "-r", "now"])
+    expect(reboot).toEqual("");
```
Comment (imobachgs) [inline web/src/lib/client/manager.test.js:76]: We might consider resolving to a boolean, so you know whether the command was successfully run.

File: web/src/lib/client/manager.js [hunk 3]
```diff
@@ -87,6 +88,16 @@ export default class ManagerClient {
...
     });
   }
+
+  /**
+   * Returns whether calling the system reboot suceeded or not.
+   *
+   * @return {Promise.<boolean>}
+   */
+  rebootSystem() {
+    console.log("Rebooting the system");
```
Comment (dgdavid) [inline web/src/lib/client/manager.js:98]: NP: I guess this console output is not needed, although it does not hurt.

File: web/src/Overview.jsx [hunk 4]
```diff
@@ -59,11 +59,9 @@ function Overview() {
 
   const InstallButton = () => {
     return (
-      <>
-        <Button isLarge variant="primary" onClick={() => client.manager.startInstallation()}>
-          Install
-        </Button>
-      </>
+      <Button isLarge variant="primary" onClick={() => client.manager.startInstallation()}>
```
Comment (dgdavid) [inline web/src/Overview.jsx:62]: Nice catch :)

File: web/src/InstallationFinished.jsx [hunk 5]
```diff
@@ -39,21 +39,27 @@ import {
...
-  <Button isLarge variant="secondary" onClick={onRestart}>
-    Restart Installation
-  </Button>
+const Actions = ({ onRestart, onReboot }) => (
+  <>
+    <Button isLarge variant="secondary" onClick={onRestart}>
+      Restart Installation
+    </Button>
+    <Button isLarge variant="primary" onClick={onReboot}>
+      Reboot
```
Comment (dgdavid) [inline web/src/InstallationFinished.jsx:48]: NP: I'd use "Reboot now" or "Reboot system" to be more explicit, but taking into account that the screen already has an explanatory text it is ok as it is.

**Review by dgdavid — APPROVED** [review]
LGTM, just minor comments

File: web/src/lib/client/manager.js (hunk 3 above)
Comment (teclator) [reply web/src/lib/client/manager.js]: I will remove it.

**Comment by teclator** [comment]
@dgdavid just updated README installer finished screenshot, could you check it ?

**Review by dgdavid — APPROVED** [review]

**Pull request merged**